app/metrics.yaml

```diff
@@ -4538,3 +4538,57 @@ engine_tab:
       - fenix-core@mozilla.com
       - skaspari@mozilla.com
     expires: "2021-12-31"
+
+android_keystore_experiment:
+  experiment:
+    type: event
+    description: |
+      General data about the functionality/experiment.
```

Comment:
This will only be recorded in case of unexpected failures, so reword as such: "Records an instance of an unexpected failure during the experiment"